### ISSUE TITLE
[PostgreSQLError.fieldType: No PostgreSQL column type known for Dictionary<String, String>.]

### ISSUE THREAD

**Issue by abbasmousavi** [vapor/fluent-postgresql#20]
With fluent-postgresql "1.0.0-rc.1.1", When there is a swift dictionary in the model, I get this error

```
[PostgreSQLError.fieldType: No PostgreSQL column type known for Dictionary<String, String>.] [Suggested fixes: Conform Dictionary<String, String> to `PostgreSQLColumnStaticRepresentable` to specify field type or implement a custom migration. Specify the `PostgreSQLColumn` manually using the schema builder in a migration.]
```

**Comment by tanner0101** [comment]
It may be possible to do this with another method. I'll look into it again.

**Comment by tanner0101** [comment]
Ah, I remember the problem now: arrays should work fine. The problem with dictionaries is that we cannot dynamically cast the dictionary as `Codable` during runtime since 4.1 does not yet support that where conditional conformance is used. Since we can't cast the dictionary as Codable, and we also can't conditionally conform, there's no way to parse/serialize the JSON from the database. We will unfortunately need to wait until this is fully implemented by Swift in 4.2 or 5. 

Luckily arrays should work and nested structs will work in the meantime! Let me know if you have any issues with that.
